src/config/redirects.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Configure a list of redirects.
+ */
+
+import path from "path"
+
+const redirects = {
+  "/ordering-guide/": "/buyers-guide/ordering-guide/",
```

Comment:
I think we need to give this some thought given that the site is being served at `https://gsa.gov/oasis-plus/`

Maybe adding a line: 
`"/oasis-plus/ordering-guide/": "/oasis-plus/buyers-guide/ordering-guide/",` 

will work?

Reply:
my initial checkin commit is using ""/ordering-guide/": "/oasis-plus/buyers-guide/ordering-guide/", but it both not working locally or in feature branch. I can revert it back for testing purpose though, I think feature branch is not rendering properly.